### PR TITLE
Improve job definition layout and contents in a handful of dirs in the base providers (BugFix)

### PR DESCRIPTION
**Pull request by kissiel**
Another portion of changes done by the pxu-fixer.
I added the hunks by hand to do a front-line review.

The changes proposed here are/should be only cosmetic.

### CODE REVIEW

File: providers/base/units/disk/encryption.pxu [hunk 1]
```diff
@@ -23,28 +22,19 @@ estimated_duration: 2.0
...
-plugin: manual
 _purpose:
- The device partition is encrypted using TPM master key. To unseal the master
- key from TPM, PCR7 (Platform Configuration Register 7) needs to be identical
- to the value it had when the master key was sealed into TPM. Every time the
- device boots, it checks PCR7 to unseal TPM and retrieves master key from TPM
- to decrypt its data partition. If TPM PCR7 is modified (e.g. by flashing the
- BIOS), the device won't be able to get the master key and decrypt its data
- partition.
+ The device partition is encrypted using TPM master key. To ensure the security of the device, it is crucial that the data partition cannot be decrypted—and consequently, the device cannot boot—if PCR7 (Platform Configuration Register 7) is modified. This test verifies the system's resilience against unauthorized modifications by ensuring it cannot boot if the PCR7 value is altered.
```
Comment (Hook25) [inline providers/base/units/disk/encryption.pxu:26]:
```suggestion
 The device partition is encrypted using TPM master key. To unseal the master
 key from TPM, PCR7 (Platform Configuration Register 7) needs to be identical
 to the value it had when the master key was sealed into TPM. Every time the
 device boots, it checks PCR7 to unseal TPM and retrieves master key from TPM
 to decrypt its data partition. If TPM PCR7 is modified (e.g. by flashing the
 BIOS), the device won't be able to get the master key and decrypt its data
 partition. This test verifies the system's resilience against unauthorized 
 modifications by ensuring it cannot boot if the PCR7 value is altered.
```

Old purpose is mostly clearer imo

File: providers/base/units/disk/encryption.pxu [hunk 2]
```diff
@@ -23,28 +22,19 @@ estimated_duration: 2.0
...
- NOTE!!!! YOU'LL HAVE TO RE-INSTALL THE IMAGE AFTER THIS TEST
+ NOTE: YOU'LL HAVE TO RE-INSTALL THE IMAGE AFTER THIS TEST.
  1. Install the image and make sure it boots and you can log in.
- 2. Make sure the BIOS is set up properly (e.g. TPM enabled, UEFI boot mode)
- 3. Based on your project's situation, choose one of the following methods to clean TPM:
-    a. Turn the device off and upgrade/downgrade the BIOS or modify Secure Boot state
-    b. Clean TPM via BIOS menu
-    c. Install checkbox, execute "checkbox-[project name].checkbox-cli run com.canonical.certification::tpm2.0_3.0.4/tpm2_takeownership"
- 4. Start or reboot the device
+ 2. Ensure the BIOS is set up correctly (e.g., TPM enabled, UEFI boot mode).
```
Comment (Hook25) [inline providers/base/units/disk/encryption.pxu:30]:
```suggestion
 2. Ensure the BIOS is set up correctly (e.g. TPM enabled, UEFI boot mode).
```

File: providers/base/units/disk/encryption.pxu [hunk 3]
```diff
@@ -23,28 +22,19 @@ estimated_duration: 2.0
...
+ 3. Depending on your project's requirements, choose one of these methods to clean TPM:
```
Comment (Hook25) [inline providers/base/units/disk/encryption.pxu:31]:
```suggestion
 3. Depending on the DUT, choose one of these methods to clean TPM:
```

File: providers/base/units/disk/jobs.pxu [hunk 4]
```diff
@@ -143,16 +143,16 @@ requires:
...
-  Which also known as free fall sensor checking in terms of laptop PC.
- STEPS:
-  1. Click on Test
-  2. Move the system under test around, ensuring it is raised and lowered at some point.
- VERIFICATION:
-  The verification of this test is automated. Do not change the
-  automatically selected result.
+_purpose:
+ This test checks that a system's drive protection mechanism works properly.
+ Which is also known as free fall sensor checking in terms of laptop PC.
```
Comment (Hook25) [inline providers/base/units/disk/jobs.pxu:148]:
```suggestion
 This test checks that a system's drive protection mechanism (free fall
 sensor in laptops) works properly.
```

File: providers/base/units/disk/jobs.pxu [hunk 5]
```diff
@@ -143,16 +143,16 @@ requires:
...
+_steps:
+ 1. Click on Test
+ 2. Move the system under test around, ensuring it is raised and lowered at some point.
+_verification:
+ The verification of this test is automated. Do not change the
+ automatically selected result.
+_summary: Evaluate the system's drive protection mechanism, including free fall sensor functionality, through interactive testing.
```
Comment (Hook25) [inline providers/base/units/disk/jobs.pxu:155]:
```suggestion
_summary: Test the system's hard drive protection mechanism.
```
(80 char)

File: providers/base/units/dock/jobs.pxu [hunk 6]
```diff
@@ -1668,9 +1677,8 @@ command:
     else
         exit 1
     fi
-_description:
- Tests that the systems wireless hardware can connect to a router using WPA
- security and the 802.11b/g protocols after the system has been suspended.
+_purpose: Tests that the system's wireless hardware can connect to a router using WPA security and the 802.11b/g protocols after the system has been suspended.
+_summary: Ensure wireless hardware can connect using WPA and 802.11b/g after suspend.
```
Comment (Hook25) [inline providers/base/units/dock/jobs.pxu:1681]:
```suggestion
```
Duplicated

File: providers/base/units/dock/jobs.pxu [hunk 7]
```diff
@@ -2438,32 +2444,35 @@ category_id: dock
...
+    4. Connect an HDMI monitor to the docking station.
     5. Connect a Thunderbolt monitor to the docking station.
     6. Connect a Thunderbolt3 storage to the docking station.
     7. Connect a USB Type-C storage to the docking station.
     8. Connect a USB3.0 storage to the docking station.
     9. Connect a headset to the docking station.
 _verification:
     Mark it Pass if you have connected all I/O ports as mentioned above.
+_summary:
+    Verify the initial setup of all I/O ports on the docking station by connecting various devices and ensure they meet the dock's specification.
```
Comment (Hook25) [inline providers/base/units/dock/jobs.pxu:2475]:
```suggestion
    Prepare all I/O ports on the dock by connecting all devices it supports.
```

Summary is wrong and long

**Review by Hook25 — CHANGES_REQUESTED** [review]
Just one requested change (remove the duplicated field)

File: providers/base/units/disk/encryption.pxu (hunk 2 above)
Comment (kissiel) [reply providers/base/units/disk/encryption.pxu:30]: Common English style guides suggest adding a comma after "e.g." and "i.e.".

File: providers/base/units/dock/jobs.pxu [hunk 8]
```diff
@@ -1046,11 +1053,12 @@ depends: dock/cold-plug
 estimated_duration: 1.0
 command:
  if [ "$(upower -i /org/freedesktop/UPower/devices/battery_BAT0 | grep "state" | awk '{print $2}')" = "discharging" ] ; then exit 1 ; fi
-_description:
+_purpose:
  This test will make sure the laptop is being charged while connected to the dock.
 
 ### Hot Plug ###
 
+
```
Comment (fernando79513) [inline providers/base/units/dock/jobs.pxu:1061]:
```suggestion
```

File: providers/base/units/dock/jobs.pxu [hunk 9]
```diff
@@ -1188,7 +1197,7 @@ requires:
  package.name == 'alsa-base'
  package.name == 'gir1.2-gst-plugins-base-0.10' or package.name == 'gir1.2-gst-plugins-base-1.0'
  package.name in ['pulseaudio-utils', 'pipewire']
-_summary: HDMI audio test
+_summary: HDMI audio test via USB Type-C DisplayPort
```
Comment (fernando79513) [inline providers/base/units/dock/jobs.pxu:1200]:
```suggestion
_summary: HDMI audio test for USB Type-C DisplayPort
```

File: providers/base/units/dock/jobs.pxu [hunk 10]
```diff
@@ -1335,36 +1346,34 @@ _summary: Audio status before suspend
...
-_description: Tests whether the system has a working Internet connection.
+_purpose: Tests whether the system has a working Internet connection.
 
 plugin: user-interact
 id: dock/usb-insert-after-suspend
 category_id: dock-usb
 depends: suspend/suspend_advanced_auto
 estimated_duration: 10.0
-_summary: USB drive insertion after resuming
+_summary: Verify that USB drive insertion is detected after system resumes from suspend
```
Comment (fernando79513) [inline providers/base/units/dock/jobs.pxu:1366]:
```suggestion
_summary: Verify USB insertion after resuming from suspend
```

File: providers/base/units/dock/jobs.pxu [hunk 11]
```diff
@@ -1442,16 +1451,16 @@ _verification:
     The verification of this test is automated. Do not change the
     automatically selected result.
 
+
```
Comment (fernando79513) [inline providers/base/units/dock/jobs.pxu:1454]:
```suggestion
```

**Review by fernando79513 — CHANGES_REQUESTED** [review]
I tried to take a look at most of them. There is still some inconsistency about how we use `_summary` and  `_purpose` but I think this PR is an improvement overall.

I think mostly the summary should try to follow a common structure (short, all in infinitive, congruency on using test/check/assert) but I think that would be another IA pass with some clear instructions.

**Comment by Hook25** [comment]
Rebased and merged separately. Thanks

**Pull request closed without merging**